gtsfm/frontend/verifier/degensac.py

```diff
@@ -18,87 +18,63 @@
 import pydegensac
 from gtsam import Cal3Bundler, Rot3, Unit3
 
+import gtsfm.utils.logger as logger_utils
 import gtsfm.utils.verification as verification_utils
 from gtsfm.common.keypoints import Keypoints
 from gtsfm.frontend.verifier.verifier_base import VerifierBase
 
-# minimum matches required for computing the F-matrix
+NUM_MATCHES_REQ_E_MATRIX = 8  # cannot compute E matrix and hence falling back on F matrix.
```

Comment:
i didnt catch whats going on here @ayushbaid  -- why note 5?

Reply:
There is no E matrix estimation for degensac. Maybe I just remove this variable?

Reply:
yeah it seems to be confusing